interfaces/vrep/DQ_SerialVrepRobot.m

```diff
@@ -0,0 +1,240 @@
+            %     
+            %     Note that the name of the robot should be EXACTLY the same as in the CoppeliaSim
+            %     scene. For instance, if you drag-and-drop a second robot, its name will become 
+            %     "my_robot#0", a third robot, "my_robot#1", and so on.
+
+            obj.robot_name = robot_name;
+            obj.vrep_interface = vrep_interface;
+
+            %% The use of 'initialize_joint_names_from_vrep()', as is done in the C++ implementation, is not supported on a constructor in MATLAB
+            % From the second copy of the robot and onward, VREP appends a
+            % #number in the robot's name. We check here if the robot is
+            % called by the correct name and assign an index that will be
+            % used to correctly infer the robot's joint labels.
+            splited_name = strsplit(obj.robot_name,'#');
+            robot_label = splited_name{1};
+            if ~strcmp(robot_label, base_robot_name)
+                error('Expected %s', base_robot_name)
+            end
+            if length(splited_name) > 1
+                robot_index = splited_name{2};
+            else
+                robot_index = '';
+            end
+            
+            % Initialize joint names, link names, and base frame name
+            obj.joint_names = {};
+            for i=1:robot_dof
+                current_joint_name = {robot_label,'_joint',int2str(i),robot_index};
+                obj.joint_names{i} = strjoin(current_joint_name,'');
+            end
+            obj.base_frame_name = obj.joint_names{1};
+        end
+
+        function joint_names = get_joint_names(obj)
+            % This method gets the joint names of the robot in the CoppeliaSim scene.
+            % Usage:
+            %     get_joint_names;
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     joint_names = vrep_robot.get_joint_names;
+
+            joint_names = obj.joint_names;
+        end
+
+        function set_configuration_space_positions(obj, q)
+            % This method sets the joint configurations to the robot in the CoppeliaSim scene.
+            % Usage:
+            %     set_configuration_space_positions(q);  
+            %          q: The joint configurations of the robot in the CoppeliaSim scene.
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     q = zeros(7,1);
+            %     vrep_robot.set_configuration_space_positions(q);
+            %     
+            %     Note that this calls "set_joint_positions" in the remoteAPI, meaning that it
+            %     is only suitable for passive joints.
```

Comment:
Please double check if the "passive" and "active" joint terminology is what is being used in CoppeliaSim.

Reply:
Updated in [3c4d66e](https://github.com/dqrobotics/matlab/pull/108/commits/3c4d66e9f1c4d12fd2d9ec61fb1274f7c845fed6). It should probably be changed in the [C++ version](https://github.com/dqrobotics/cpp-interface-vrep/blob/master/src/dqrobotics/interfaces/vrep/DQ_SerialVrepRobot.cpp) as well.